hedera-mirror-importer/src/main/resources/db/migration/v1/V1.46.0__entity_timestamp.sql

```diff
@@ -0,0 +1,17 @@
+-------------------
+-- Fill missing entity created_timestamp and modified_timestamp
+-------------------
+
+with entity_timestamp as (
```

Comment:
nit: add a comment to highlight the hardcoded numbers being
e.g.
```suggestion
-- update entity rows based on successful entity create/update transaction types
with entity_timestamp as (
```